minibroker/minibroker-master.yaml

```diff
@@ -0,0 +1,272 @@
+---
+shared:
+  helm-charts-s3-configs: &helm-charts-s3-configs
+    access_key_id: ((s3-access-key))
+    bucket: ((s3-bucket))
+    disable_ssl: ((s3-disable-ssl))
+    endpoint: ((s3-endpoint))
+    private: false
+    secret_access_key: ((s3-secret-key))
+  helm-charts-src-configs: &helm-charts-src-configs
+    branch: ((helm-charts-src-branch))
+    private_key: ((github-private-key))
+    uri: ((helm-charts-src-repo-ref))
+  s3-configs: &s3-configs
+    access_key_id: ((s3-access-key))
+    bucket: ((s3-bucket))
+    disable_ssl: ((s3-disable-ssl))
+    endpoint: ((s3-endpoint))
+    private: false
+    secret_access_key: ((s3-secret-key))
+  src-ci-configs: &src-ci-configs
+    uri: ((ci-repo))
+    branch: ((ci-branch))
+    private_key: ((github-private-key))
+
+
+resources:
+- name: src
+  type: git
+  source:
+    branch: ((src-branch))
+    private_key: ((github-private-key))
+    uri: ((src-repo-ref))
+
+- name: src-redis
+  type: git
+  source:
+    <<: *helm-charts-src-configs
+    paths:
+    - stable/redis
+
+- name: src-postgres
+  type: git
+  source:
+    <<: *helm-charts-src-configs
+    paths:
+    - stable/postgresql
```

Comment:
Huh, it's `postgres` in PRs and `postgresql` in `master`?

Reply:
Fixed in 4a85c51